README.md

```diff
@@ -1055,24 +1076,27 @@ when the stock is fully owned—still, this is actually one of the least common
 equity. We mention it first because it is the simplest form of equity compensation, useful
 for comparison as things get more complex.
 
-- 🄳 A **restricted stock award** is when a company grants someone stock as a form of
-  compensation. The stock awarded has additional conditions on it, including a vesting
-  schedule, so is called **restricted stock**. Restricted stock awards may also be called
-  simply **stock awards** or **stock grants**.
-- ∑ What *restricted* means here is actually
-  [complex](https://github.com/jlevy/og-equity-compensation/issues/24). It refers to the fact
-  that the stock (i) has certain restrictions on it (like transfer restrictions) required
-  for private company stock, and (ii) will be subject to repurchase at cost pursuant to a
-  vesting schedule.
-  The repurchase right lapses over the service-based vesting period, which is what is meant
-  in this case by the stock “vesting.”
-- ☝️ Restricted stock awards are
-  [not the same thing](https://www.fool.com/knowledge-center/the-difference-between-a-restricted-stock-unit-res.aspx)
-  as [restricted stock units](#rsus).
-- Typically, stock awards are limited to executives or very early hires, since once the
-  value of the shares increases, the tax burden of receiving them (without paying the
-  company for their value) can be too great for most people.
-  Usually, instead of restricted stock, an employee will get stock options.
+🄳 A **restricted stock award** is when a company grants someone stock as a form of
+compensation. The stock awarded has additional conditions on it, including a vesting
+schedule, so is called **restricted stock**. Restricted stock awards may also be called
+simply **stock awards** or **stock grants**.
+
+∑ What *restricted* means here is actually
+[complex](https://github.com/jlevy/og-equity-compensation/issues/24). It refers to the fact
```

Comment:
So the initial description says this is the simplest form of equity, and now it's "actually complex." Again, considering that this description is very technical and uses a lot of terminology that we don't explain until we get to stock options, consider moving to "Less common types of equity."